examples/fluids/advection.h

```diff
@@ -59,25 +58,59 @@ struct DCContext_ {
 };
 #endif
 
+#ifndef surface_context_struct
+#define surface_context_struct
+typedef struct SurfaceContext_ *SurfaceContext;
+struct SurfaceContext_ {
+  CeedScalar E_wind;
+  CeedScalar strong_form;
+  bool implicit;
+};
+#endif
+
 // *****************************************************************************
 // This QFunction sets the the initial conditions and boundary conditions
+//   for two test cases: ROTATION and TRANSLATION
+//
+// -- ROTATION (default)
+//      Initial Conditions:
+//        Mass Density:
+//          Constant mass density of 1.0
+//        Momentum Density:
+//          Rotational field in x,y
+//        Energy Density:
+//          Maximum of 1. x0 decreasing linearly to 0. as radial distance
+//            increases to (1.-r/rc), then 0. everywhere else
+//
+//      Boundary Conditions:
+//        Mass Density:
+//          0.0 flux
+//        Momentum Density:
+//          0.0
+//        Energy Density:
+//          0.0 flux
 //
-// Initial Conditions:
-//   Mass Density:
-//     Constant mass density of 1.0
-//   Momentum Density:
-//     Rotational field in x,y with no momentum in z
-//   Energy Density:
-//     Maximum of 1. x0 decreasing linearly to 0. as radial distance increases
-//       to (1.-r/rc), then 0. everywhere else
+// -- TRANSLATION
+//      Initial Conditions:
+//        Mass Density:
+//          Constant mass density of 1.0
+//        Momentum Density:
+//           Constant rectilinear field in x,y
+//        Energy Density:
+//          Maximum of 1. x0 decreasing linearly to 0. as radial distance
+//            increases to (1.-r/rc), then 0. everywhere else
 //
-//  Boundary Conditions:
-//    Mass Density:
-//      0.0 flux
-//    Momentum Density:
-//      0.0
-//    Energy Density:
-//      0.0 flux
+//      Boundary Conditions:
+//        Mass Density:
+//          0.0 flux
+//        Momentum Density:
+//          0.0
+//        Energy Density:
+//          Inflow BCs:
+//            E = 1E6 is applied weakly in QFunction "Advection2d_Sur"
+//          Outflow BCs:
+//            The current values of E are applied weakly in
+//              QFunction "Advection2d_Sur"
```

Comment:
Thanks @LeilaGhaffari Leila for updating these descriptions. What do you think of the following suggestion? Not sure if the outflow one is correct this way. But I thought the description of the QFunction could have been combined for both and put afterwards.
```suggestion
//          Inflow BCs:
//            E = E_wind
//          Outflow BCs:
//            E = E(boundary)
//           Both In/Outflow BCs for E are applied weakly in the QFunction "Advection2d_Sur"
```

Reply:
Same thing would apply to `advection_2d.h`.

Overall everything works and looks good to me. Great job!

Reply:
@valeriabarra , it is correct and much cleaner. Thanks. It's fixed.